sdk/Sdk/Constants.cs

```diff
@@ -44,9 +44,11 @@ internal static class Constants
         // NetFramework
         internal const string NetCoreApp31 = "netcoreapp3.1";
         internal const string Net60 = "net6.0";
```

Comment:
should we just remove these constants `Net60` and `NetCoreVersion6` as we are most likely not using them anywhere in the code?